src/test/java/com/example/ludogoriesoft/lukeriaerpapi/services/ClientServiceTest.java

```diff
@@ -104,20 +104,6 @@ void testGetClientById_NonExistingId() {
         verifyNoInteractions(modelMapper);
     }
 
-    @Test
```

Comment:
Because I had a problem with them and I saw that in sonar we cover over 90% of this branch and it was more practical for me to delete them after I couldn't fix them. But I will try to fix them again

Reply:
**Great work!**